baseapp/baseapp.go

```diff
@@ -117,7 +120,8 @@ func (app *BaseApp) SetCommitMultiStoreTracer(w io.Writer) {
 	app.cms.WithTracer(w)
 }
 
-// Mount IAVL or DB stores to the provided keys in the BaseApp multistore
+// MountStores mounts all IAVL or DB stores to the provided keys in the BaseApp
+// multistore.
 func (app *BaseApp) MountStores(keys ...*sdk.KVStoreKey) {
```

Comment:
Can we make this MountStores function take both transient keys and store keys? It could look like:

```go
func (app *BaseApp) MountStores(keys ...interface{}) {
  for _, key := range keys {
    switch key.(type) {
    case *sdk.KVStoreKey:
	if !app.fauxMerkleMode {
		app.MountStore(key, sdk.StoreTypeIAVL)
	} else {
		// StoreTypeDB doesn't do anything upon commit, and it doesn't
		// retain history, but it's useful for faster simulation.
		app.MountStore(key, sdk.StoreTypeDB)
	}
    case *sdk.TransientStoreKey:
      app.MountStore(key, sdk.StoreTypeTransient
    }
  }
}
```

Reply:
I think this is a good idea - but perhaps best for a separate PR?

Reply:
Yeah. I think this falls under general `app.go` cleanup